Sources/Operators.swift

```diff
@@ -16,7 +16,7 @@ import Foundation
 
 // MARK: - <| Operator
 
-infix operator <| { associativity left precedence 150 }
+infix operator <|
```

Comment:
Looks like maybe it's from this proposal? https://github.com/apple/swift-evolution/blob/master/proposals/0077-operator-precedence.md